README.adoc

```diff
@@ -413,13 +413,18 @@ Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 5.92 s - in it.i
 Running it.io.openliberty.guides.system.SystemEndpointIT
 Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.017 s - in it.io.openliberty.guides.system.SystemEndpointIT
 Running it.io.openliberty.guides.inventory.InventoryEndpointIT
+[WARNING ] Interceptor for {http://client.inventory.guides.openliberty.io/}SystemClient has thrown exception, unwinding now
+Could not send Message.
+[err] The specified host is unknown.
 Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.077 s - in it.io.openliberty.guides.inventory.InventoryEndpointIT
 
 Results:
 
 Tests run: 7, Failures: 0, Errors: 0, Skipped: 0
 ----
 
+The warning message is expected and results from a request to a bad or an unknown host name.
```

Comment:
From reviews of other similar changes, minor change to say "The warning messages and errors are expected" and to mention which test causes them.